_source/logzio_collections/_contributors/yotamloe.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Yotam Loewenbach
+github: yotamloe
+
```

Comment:
linkedin: www.linkedin.com/in/yotam-loewenbach
twitter: @YLoewenbach

Reply:
Added, will be in next push